manifests/0000_10_config-operator_01_authentication.crd.yaml

```diff
@@ -79,9 +452,15 @@ spec:
                       name:
                         description: name is the metadata.name of the referenced secret
                         type: string
+                    required:
+                    - name
                     type: object
+                required:
+                - kubeConfig
                 type: object
               type: array
+          required:
+          - type
```

Comment:
@damemi why is this required if it has no required tag in https://github.com/openshift/api/blob/master/config/v1/types_authentication.go#L29 ?